modules/nw-customize-ingress-error-pages.adoc

```diff
@@ -0,0 +1,96 @@
+
+Custom error code response pages are specified in a config map then patched to the Ingress Controller. The config map keys have two available file names as follows:
+`error-page-503.http` and `error-page-404.http`.
+
+Custom http error code response pages should follow the link:https://www.haproxy.com/documentation/hapee/latest/configuration/config-sections/http-errors/[HAProxy http error page configuration guidelines]. Here is an example of the default {product-title} HAProxy router link:https://raw.githubusercontent.com/openshift/router/master/images/router/haproxy/conf/error-page-503.http[http 503 error code response page].
+
+By default, {product-title} router serves only a 503 error page when the server is not running or when the route is incorrect or non-existent. This default behavior is the same as the behavior on {product-title} 4.8 and earlier. If a config map for the customization of an http error code response is not provided, and you are using a custom http error code response page, the router serves a default 404 or 503 error code response page for the http error code response whose customization was not provided in the config map.
+
+[NOTE]
+====
+If you use the {product-title} default 503 error code page as a template for your customizations, the headers in the file require an editor than can use CRLF line endings.
+====
+
+.Procedure
+
+. Create a config map named "my-custom-error-code-pages" in the `openshift-config` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-config create configmap my-custom-error-code-pages \
+--from-file=error-page-503.http \
+--from-file=error-page-404.http
+----
+
+. Patch the Ingress Controller to reference the `my-custom-error-code-pages` config map by name:
++
+[source,terminal]
+----
+$ oc patch -n openshift-ingress ingresscontroller/default --patch '{"spec":{"httpErrorCodePages": "my-custom-error-code-pages"}}' --type=merge
+----
+
+. The ingress operator copies the `my-custom-error-code-pages` config map from the `openshift-config` namespace to the `openshift-ingress` namespace. A configmap with name format `<your-ingresscontroller-name>-errorpages` gets created in the `openshift-ingress` namespace. You can see the copy by running the following command:
++
+[source,terminal]
+----
+$ oc get cm default-errorpages -n openshift-ingress
+NAME                       DATA   AGE
+default-errorpages         2      25s  <1>
+----
+<1> The name `default-errorpages` because in Step 2, the default ingress controller is patched.
++
+The config map containing the custom error pages mounts on the router volume where the config map key is the filename that has custom http error code response.
```

Comment:
Sigh, yes this detail seems extraneous and not really helpful for the customer. Agree 

Reply:
@mikemckiernan  we need to tell the customer where this configmap is mounted. The operator scales a router deployment with this configmap details and then the router mounts this configmap on router volume.

Reply:
If folks prefer to keep it, please revise so that it is clearer that the router pod performs the mount rather than--what I believe the current sentence suggests--that the config map mounts on the router volume. 

Please also upcase HTTP.